lib/davinci_pas_test_kit/custom_groups/v2.0.1/pas_client_registration_group.rb

```diff
@@ -0,0 +1,125 @@
+        endpoints.
+
+        When running these tests there are 4 options for authentication:
+        1. **UDAP B2B client credentials flow**: the system under test will dynamically register
+           with Inferno and request access tokens used to access FHIR endpoints
+           as per the UDAP specification.
+        2. **SMART Backend Services**: the system under test will manually register
+           with Inferno and request access token used to access FHIR endpoints
+           as per the SMART Backend Services specification.
+        3. **Both UDAP and SMART**: the system under test will register as both a SMART
+           and a UDAP client, using the same client id for both. If this path is chosen
+           both authentication approaches will need to be demonstrated during the
+           subsequent tests.
+        4. **Dedicated Endpoints**: Inferno will create a dedicated set of FHIR endpoints for this session
+           so that the system under test does not need to get access tokens or provide
+           them during these tests. Since PAS requires authentication of client systems,
+           testers will be asked to attest that their system supports another form of
+           authentication, such as mutual authentication TLS.
+      )
+      run_as_group
+
+      input :udap_client_uri,
+            title: 'UDAP Client URI',
+            type: 'text',
+            optional: true,
+            description: %(
+              Required if demonstrating the UDAP B2B client credentials flow. Must be the
+              Client URI that will be used to register with Inferno's simulated UDAP server.
+            )
+      input :smart_jwk_set,
+            title: 'SMART JSON Web Key Set (JWKS)',
+            type: 'textarea',
+            optional: true,
+            description: %(
+              Required if demonstrating the SMART backend services flow. May be either
+              a publicly accessible url containing the JWKS, or the raw JWKS.
+            )
+      input :client_id,
+            title: 'Client Id',
+            type: 'text',
+            optional: true,
+            description: %(
+              If demonstrating only the SMART backend
+              services flow and a particular client id is desired, put it here. If
+              demonstrating the UDAP B2B client credentials flow as well, any value placed
+              here will be overriden and the same client id as generated for UDAP will be used.
+            )
+      input :session_url_path,
+            title: 'Session-specific URL path extension',
+            type: 'text',
+            optional: true,
+            description: %(
+              Ignored if a UDAP Client URI or SMART JWKS is provided. If demonstrating PAS
+              interactions without authentication, Inferno will use this value to
+              setup session-specific FHIR endpoints to use during these tests.
```

Comment:
Just for the sake of consistent verbiage throughout for clarity
```suggestion
              setup dedicated session-specific FHIR endpoints to use during these tests.
```

Reply:
fixed in [e11f366](https://github.com/inferno-framework/davinci-pas-test-kit/pull/27/commits/e11f36633d2602306926e8bacc307c71e69a336c)